app/controllers/invitations_controller.rb

```diff
@@ -0,0 +1,21 @@
+class InvitationsController < ApplicationController
+  def create
+    invitation = Invitation.new(model_param)
+
+    if invitation.create_and_send
+      redirect_to(
+        root_path,
+        notice: "邀請信已經送到您的信箱囉！很期待見到你！")
+    else
+      redirect_to(
+        root_path,
+        alert: "唉呦！出錯了... #{invitation.errors.full_messages}.")
+    end
+  end
+
+  private
+
+  def model_param
```

Comment:
Personally I do not like extra indent for private stuffs because it's
not really one extra layer and in the end it would end up like this if
we indent one extra level:

    class C
      private
        def f
        end
    end

This looks quite weird to me as we're now ending two levels at once. I
saw people might do this in this case:

    class C
    private
      def f
      end
    end

This looks better to me. Personally I always do this though:

    class C
      private
      def f
      end
    end

I've been doing this from the beginning.